dataactcore/interfaces/function_bag.py

```diff
@@ -327,4 +327,27 @@ def markFileComplete(job_id, filename=None):
     sess = GlobalDB.db().session
     fileComplete = createFileIfNeeded(job_id, filename)
     fileComplete.file_status_id = FILE_STATUS_DICT['complete']
-    sess.commit()
\ No newline at end of file
+    sess.commit()
+
+def getErrorMetricsByJobId(job_id, include_file_types=False, severity_id=None):
+    """ Get error metrics for specified job, including number of errors for each field name and error type """
+    sess = GlobalDB.db().session
+    result_list = []
+
+    query_result = sess.query(File).options(joinedload("file_status")).filter(File.job_id == job_id).one()
```

Comment:
I think that, for now, we can leave it as-is. It's about a 50/50 split of needing file_status and not needing it so I wouldn't waste the extra processing power lazily adding it every time. Maybe in the future it would be useful but for now I'd value speed over the few times we have to add that extra option in text